eiffel-gerrit-plugin-script

```diff
@@ -40,12 +42,14 @@ function do_rebuild {
   echo "######### BUILDING JARS ################"
   echo "########################################"
 
+  JAR_LOCATION=$(ls target/ | grep '^eiffel-gerrit-plugin-[0-9]*.[0-9]*.[0-9]*.jar')
   mvn clean install package -DskipTests
 
+  mv target/test-classes/env.bash target/env.bash
```

Comment:
Why not copy these files directly from src/main/docker/... ?
Now Dockerfile is dependent on mvn goals that need to be executed first before build.
Maybe we want to be able to build based on released gerrit plugin jar files.

Reply:
Good point but I think that is better left for another issue since it might require larger changes in the script file. We only changed what was needed to make the script work again since it wasn't propertly transferring the plugin file to the container and to set up the needed containers to test with remrem.

Reply:
I suppose that docker is running from the target folder. Its not possible to go "../" in that context. That would be my guess why it isn't taking it from the src/main/docker.

Reply:
We run Docker build from git root in other project as context path, why not do the same here.

Reply:
I still suggest to make an issue about improving the test/start script since it's not directly relevant to this PR.

Reply:
I made an issue https://github.com/eiffel-community/eiffel-gerrit-plugin/issues/30
I'm not sure about the details since it's not my suggestion... I just created it.

Reply:
Ok

Reply:
I think this is fixed.